randomGuess.js

```diff
@@ -0,0 +1,95 @@
+const readline = require('readline');
+
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+	return new Promise((resolve, reject) => {
+		rl.question(questionText, resolve);
+	});
+}
+
+async function guessFcn(maxGuess, minGuess) {
+  //funciton takes the changing min and max and makes a random guess within the parameters
+	const maxNumber = Number.parseFloat(maxGuess);
+
+	const minNumber = Number.parseFloat(minGuess);
+
+	const difference = maxNumber - minNumber;
+
+	const random = Math.random();
+
+	const product = Number.parseFloat(random * difference);
+
+	const rawGuess = Number(product) + Number(minGuess + 1);
+
+	let guess = Math.floor(rawGuess);
```

Comment:
Instead of using these intermediate variables, you can use a function like one we made in class that generates a random number and can be done with one call.
```js
let minNum = 1
let maxNum = 100
let guess = generateRandomNumber(min, max)
```
You can see an example of that [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#Getting_a_random_integer_between_two_values_inclusive).